### DISCUSSION TITLE
Any way to reverse account migrations

### DISCUSSION THREAD

**Discussion by hugogameiro** [mastodon/mastodon#20401] [Q&A]
I was asked if after a user migrates from a server to another, if there was any way to move back to initial account.

In the documentation, it says that "[it will also irreversibly force everyone to unfollow your current account and follow your new account](https://docs.joinmastodon.org/user/moving/#move)". 

I am assuming that 'irreversible' on that sentence means that it's not possible but just wanted to double check. Thanks.

**Comment by Liquidream** [comment]
I would also really like to know the answer to this.
I'm about to move from .social to a "quieter" server. But would be nice to that, later down the line, i might still have the option to move back. 🤔
(Otherwise, sounds like my username will forever stay reserved on the largest instance, and completely unusable to me 🫤)

**Comment by cosmicexplorer** [comment]
When I tried to do this just now I had people [report they get a "not allowed" error](https://fosstodon.org/@jni/111814427312087698) when attempting to follow account A (the one I'm moving back to), and one instance admin mentioned that servers get confused by the redirect "loop" (even though I cancelled the redirect from A -> B before adding the new redirect from B -> A).

**Is there a way to force invalidation of a previous account redirect (A -> B) across mastodon servers so that I can complete my migration back to the old account A?** Context at https://circumstances.run/@hipsterelectron/111814528572965939. I'm confused why the "cancel redirect" operation fails to do this itself.

EDIT: one user has reported that while their initial attempt to follow failed, it now succeeds (https://social.treehouse.systems/@chinmay/111814787431399009), which makes me think that the A -> B redirection cancellation is being performed asynchronously somehow and is taking some time to propagate. **Note that I have also cancelled the B -> A redirect for now, although I do not know if that has affected anything.** I am going to wait a few days and see if the issue with new follows for account A resolves itself completely, and if so then I expect that trying to move my account from B -> A again in 30 days will succeed.

**Reply by meganlfox** [reply]
Did this end up working for you?

I'm also someone considering this same basic thing. Migrated A->B ages ago, thinking of migrating back B->A.
